Makefile

```diff
@@ -5,10 +5,17 @@
 ####
 
 RUN= # used by go tests to decide which tests to run (i.e. passed to -run)
-# Label it w the go version we bundle in:
-export VERSION_ADDITIONAL = -$(shell git log --pretty=format:%H | head -n 1)
-LD_FLAGS = -X github.com/pachyderm/pachyderm/src/client/version.AdditionalVersion=$(VERSION_ADDITIONAL)
-export GC_FLAGS = "all=-trimpath=${PWD}"
+
+# Don't set the version to the git hash in CI, as it breaks the go build cache.
+ifdef CIRCLE_BRANCH
+	export VERSION_ADDITIONAL = "ci_build"
```

Comment:
Instead of setting `CIRCLE_BRANCH=1` in testfaster, and having that change the `VERSION_ADDITIONAL` here, I think you could just set `VERSION_ADDITIONAL=ci_build` in testfaster directly and it would have the same effect (and I think wouldn't need any special handling)

Reply:
done this

Reply:
Reverted this as it broke things